synapse/cores/common.py

```diff
@@ -2743,15 +2754,19 @@ def _normTufoProps(self, form, props, tufo=None):
 
             valu = props.get(name)
 
-            prop = form + ':' + name
+            if name in self.uniprops:
+                prop = name
+
+            else:
```

Comment:
this feels a bit like a hack, but it'll get us one step closer.  most likely, we'll ultimately end up needing to do away with relative prop use at the API level.